src/test/groovy/com/athaydes/spockframework/report/internal/HtmlReportCreatorSpec.groovy

```diff
@@ -273,4 +298,18 @@ class HtmlReportCreatorSpec extends ReportSpec {
         }
     }
 
+    @Category( SpockReportExtension )
+    class ShowCodeBlocksEnabled {
+        static final htmlReportCreator = new HtmlReportCreator()
+        SpecInfoListener createListener() {
+            setShowCodeBlocks( true )
+            configReportCreator htmlReportCreator
+            new SpecInfoListener( htmlReportCreator )
+        }
+    }
+
+    @Category( SpockReportExtension )
+    class ShowCodeBlocksDisabled {
```

Comment:
I've tested your code, seems to work well! Merged the PR.
I will give a few last touches... there was a comment about the 3 varieties of the method to write the code, but that's very minor, I will change and show you how I think it would've been better...

Anyway, I should be able to release very soon!